src/devworkspace/index.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2018-2020 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import { devWorkspaceToDevfile } from '@eclipse-che/devworkspace-client';
+
+/**
+ * Convert a devworkspace to something that the natively dashboard understands
+ * @param devworkspace The devworkspace that you want to convert
+ */
+export function convertDevWorkspaceV2ToV1(devworkspace: any): che.Workspace {
```

Comment:
Please try to avoid using the `any` type whenever it's possible:

```javascript
export function convertDevWorkspaceV2ToV1(devworkspace: IDevWorkspace): che.Workspace {
```

Then you can create a new workspace object and add the necessary fields:

```javascript
const cheWorkspace = {} as che.Workspace;
cheWorkspace.namespace = devworkspace.metadata.namespace;
...
```